src/components/Table.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+
+class Table extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <table className="table">
+        <tr>
+          <td>AC</td>
+          <td>+/-</td>
+          <td>%</td>
+          <td>+</td>
+        </tr>
+        <tr>
+          <td>7</td>
+          <td>8/-</td>
+          <td>9</td>
+          <td>*</td>
+        </tr>
+        <tr>
+          <td>4</td>
+          <td>5/-</td>
+          <td>6</td>
+          <td>-</td>
+        </tr>
+        <tr>
+          <td>1</td>
+          <td>2/-</td>
```

Comment:
- [ ] You are yet to fix the numbers issue as pointed out earlier. Kindly do that.

  ![ddl](https://user-images.githubusercontent.com/54848363/188941581-eda0255a-9565-431a-bb5f-8a167139b9d9.jpg)

Reply:
Please kindly check it is wash changed.